uavcan/node.py

```diff
@@ -190,11 +190,7 @@ def remove_handlers(self, uavcan_type):
     def call_handlers(self, transfer):
         for uavcan_type, wrapper in self._handlers:
             if uavcan_type == get_uavcan_data_type(transfer.payload):
-                # noinspection PyBroadException
-                try:
-                    wrapper(transfer)
-                except Exception:
-                    logger.error('Transfer handler exception', exc_info=True)
+                wrapper(transfer)
```

Comment:
This change doesn't look correct. With your new logic, the first failing handler will prevent the library from invoking other handlers, which defeats the purpose of catching the exception in the library. Unless I misunderstood your intent, the old logic should be restored, with a try/except in every iteration.

Reply:
Oh yeah I did not understand the original logic correctly. Pushed a fix.